### PR TITLE
feat: dashboard accurately display on/offline peers

### PR DESCRIPTION
**Pull request by EthnTuttle**
Checks consensus numbers for offline peers, surfacing it to the user/Guardian.

Run `mprocs -c mprocs.yml`
Get to dashboard. Observe 2/2.
Run in a shell `docker stop ui-fedimintd_2-1`
Wait, refresh localhost:3000, and observe 1/2

Colors coming soon.

Work toward: https://github.com/fedimint/ui/issues/94

### CODE REVIEW

**Review by okjodom — APPROVED** [review]

File: apps/guardian-ui/src/components/AdminHeader.tsx [hunk 1]
```diff
@@ -40,10 +41,23 @@ interface AdminHeaderProps {
...
 }
 
-export const AdminHeader: FC<AdminHeaderProps> = ({ connectionCode }) => {
+export function AdminHeader({ connectionCode }: AdminHeaderProps) {
+  const [guardians, setGuardians] = useState<string | undefined>();
   const {
     state: { configGenParams },
   } = useSetupContext();
+  const { api } = useAppContext();
+  // will have to do polling
```
Comment (okjodom) [inline apps/guardian-ui/src/components/AdminHeader.tsx:50]: ack

File: apps/guardian-ui/src/components/AdminHeader.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ interface PillProp {
   status: string;
 }
 
+// Have to update pull color based on quorum.
```
Comment (okjodom) [inline apps/guardian-ui/src/components/AdminHeader.tsx:12]: ack

**Review by okjodom — APPROVED** [review]

File: apps/guardian-ui/src/components/AdminHeader.tsx [hunk 3]
```diff
@@ -40,10 +41,23 @@ interface AdminHeaderProps {
...
   const {
     state: { configGenParams },
   } = useSetupContext();
+  const { api } = useAppContext();
+  // will have to do polling
+  useEffect(() => {
+    async function getStatus() {
+      const { consensus } = await api.status();
+      const online = consensus ? consensus.peers_online + 1 : 1;
+      const offline = consensus ? consensus.peers_offline : 1;
```
Comment (okjodom) [inline apps/guardian-ui/src/components/AdminHeader.tsx:55]: actually, for a 1/1 federation, we cannot be both online and offline at the same time. Plus, should we be checking `status.server` in that case? hmm

File: apps/guardian-ui/src/components/AdminHeader.tsx (hunk 2 above)
Comment (EthnTuttle) [reply apps/guardian-ui/src/components/AdminHeader.tsx:12]: I'll confirm

**Pull request merged**